Formula/micromamba.rb

```diff
@@ -0,0 +1,83 @@
+class Micromamba < Formula
+  desc "Fast Cross-Platform Package Manager"
+  homepage "https://github.com/mamba-org/mamba"
+  url "https://github.com/mamba-org/mamba/archive/refs/tags/2023.02.03.tar.gz"
+  version "1.3.0"
+  sha256 "4c794246dd6008caebebef8c6266e6ba3f9ff5ce48908c830e448811587e9921"
+  license "BSD-3-Clause"
+
+  depends_on "cmake" => :build
+  depends_on "cli11"
+  depends_on "fmt"
+  depends_on "libsolv"
+  depends_on "libssh2"
+  depends_on "lz4"
+  depends_on "nlohmann-json"
+  depends_on "openssl@1.1"
+  depends_on "reproc"
+  depends_on "spdlog"
+  depends_on "tl-expected"
+  depends_on "xz"
+  depends_on "yaml-cpp"
+  depends_on "zstd"
```

Comment:
No linkage for
```
cli11
libssh2
nlohmann-json
spdlog
tl-expected
```
It's possible that some, or all, of these are needed only at build-time instead of runtime. `tl-expected` in particular looks to be header-only, so unlikely to be needed at runtime. Same for `nlohmann-json`.

Reply:
Actually, I think `libssh2` is needed only if you want a statically-linked `[lib]mamba`.